src/OpenColorIO/ops/fixedfunction/FixedFunctionOpData.cpp

```diff
@@ -341,7 +362,51 @@ FixedFunctionOpDataRcPtr FixedFunctionOpData::clone() const
 
 void FixedFunctionOpData::validate() const
 {
-    if(m_style==REC2100_SURROUND_FWD || m_style == REC2100_SURROUND_INV)
+    if(m_style==ACES_GAMUTMAP_13_FWD || m_style == ACES_GAMUTMAP_13_INV)
+    {
+        if (m_params.size() != 7)
+        {
+            std::stringstream ss;
+            ss  << "The style '" << ConvertStyleToString(m_style, true)
+                << "' must have seven parameters but "
+                << m_params.size() << " found.";
+            throw Exception(ss.str().c_str());
+        }
+
+        const double lim_cyan = m_params[0];
+        const double lim_magenta = m_params[1];
+        const double lim_yellow = m_params[2];
+        const double thr_cyan = m_params[3];
+        const double thr_magenta = m_params[4];
+        const double thr_yellow = m_params[5];
+        const double power = m_params[6];
+
+        auto check_bounds = [](const std::string & name, double val, double low, double high) {
```

Comment:
The coding style puts curly brackets on a line.

Reply:
Fixed.